apps/package.json

```diff
@@ -152,7 +152,7 @@
     "react": "15.3.2",
     "react-addons-test-utils": "15.3.0",
     "react-bootstrap": "0.30.1",
-    "react-color": "2.2.5",
+    "react-color": "^2.11.7",
```

Comment:
Yeah, I figured as much - when you `yarn upgrade` a pinned version the default behavior is to replace with with a caret-version, probably for exactly the reasons you stated.